src/cascade/executor/epiviz_runner.py

```diff
@@ -104,21 +106,46 @@ def add_omega_constraint(model_context, execution_context):
 
 
 def model_context_from_settings(execution_context, settings):
-    model_context = initial_context_from_epiviz(settings)
+    """
+     1. Freeze the measurement bundle, which means that we make a copy of the
+        input measurement data and study covariates for safe-keeping
+        in case there is later question about what data was used.
 
-    fixed_effects_from_epiviz(model_context, execution_context, settings)
-    random_effects_from_epiviz(model_context, settings)
+     2. Retrieve the measurement bundle and its study covariates
+        and convert it into data on the model, as described in
+        :ref:`convert-bundle-to-measurement-data`.
+
+     3. Add mortality data. This is cause-specific mortality
+        data, and it is added as "mtspecific" in the Dismod-AT measurements.
+        This data has a "measurement upper" and "measurement lower"
+        which are converted into a standard error with a Gaussian
+        prior.
+
+     4. Add other-cause mortality as a constraint on the system, meaning
+        the age-standardized death rate is used to construct both
+        measurement data for mtall, with priors determined from
+        "measurement upper" and "measurement lower", but also a constraint
+        on omega, the underlying rate for other-cause mortality, so that
+        Dismod-AT will accept this as a given in the problem.
+
+     5. Create Average Integrand Cases, which are the list of
+        desired outputs from Dismod-AT to show in graphs in EpiViz-AT.
+
+     6. Construct all Fixed Effects.
+
+     7. Construct all Random Effects.
```

Comment:
and more detail here.

Reply:
added a link to the definition of the model variables

Reply:
I'm weighing what should be close to the code and what should be far. I'll argue for these two docstrings on the grounds that they are the two highest-level and most important functions in the whole application.

Reply:
I question any comment that obscures what is actually happening. If we're going to have high level step-by-steps like this in the code then I think they go at module level? At least they are out of the way.